README.md

```diff
@@ -20,6 +20,12 @@ Add [![Gem Version][gem-badge]][gem] to Gemfile:
 gem 'rails_email_preview', '~> 2.2.3'
 ```
 
+This gem requires a Sass engine, please ensure you have **one** of these gems in your Gemfile:
```

Comment:
On top of this file where it says "Compatible with Rails 4.2+", please change to "Compatible with Rails 6.1+"